hw/ip/dcd/data/dcd.hjson

```diff
@@ -0,0 +1,402 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "dcd",
+  clock_primary: "clk_i",
+  other_clock_list: [
+    "clk_aon_i",
+  ],
+  reset_primary: "rst_ni",
```

Comment:
you probably need a reset for `clk_aon_i` domain don't you?

Reply:
ah i see you have a `rst_slow_ni` later.  You'll also need to include it here in the other reset list.
You can look at `usbdev` or `pinmux` as examples.

Reply:
Re: verible installation. 

I tried to find the Ubuntu version in my desktop. It's not clear to me which version I should download/install.

eshiu@leopard:~$ lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux rodete
Release:        rodete
Codename:       rodete

Reply:
ah please have a look at the install instructions [here](https://docs.opentitan.org/doc/ug/install_instructions/#verible). It's another open source tool.

Reply:
yes the version that is used in the installation guide that @tjaychen linked should work on rodete.